src/app/shared/components/slide-open-menu/slide-open-menu.component.scss

```diff
@@ -0,0 +1,42 @@
+.slide-open-menu-container {
+    @apply m-1;
+}
+
+.slide-open-menu-header {
+    @apply flex;
+    @apply cursor-pointer;
+}
+
+.slide-open-menu-arrow-container {
+    @apply w-6 h-6 mt-1;
+    @apply text-gray-600;
+    @apply outline-none;
+
+    @apply focus:outline-none;
+}
+
+.slide-open-menu-arrow-image {
+    @apply w-4 h-4;
+    @apply rotate-0;
+    @apply transition-all duration-200 transform;
+}
+
+.rotate-arrow {
+    ::ng-deep svg {
+        @apply rotate-180;
+    }
+}
+
+.slide-open-menu-content {
+    @apply flex flex-col;
+    @apply ml-4;
+    @apply overflow-hidden;
+
+    max-height: 0;
+    transition: max-height 500ms ease-out;
+}
+
+.show-content {
+    max-height: 100px;
+    transition: max-height 200ms ease-in;
+}
```

Comment:
I realise now just how much of a pain it is to get a sliding menu like this working. I couldn't get it even remotely working with tailwind so I just used pure CSS. I also set the target height to 100px, which is obviously problematic if the menu ends up taller. I really wasn't sure how else to solve the problem.